docs/index.md

```diff
@@ -43,6 +57,27 @@ core_lib:
         protocol: sqlite
   ...
 ```
+#### `your_core_lib.yaml` Explained:
+<b>your_core_lib.yaml is the setting for your entire Core-Lib library. The above example will show how to configure core-lib to connect to a database using [SQLAlchemy](https://docs.sqlalchemy.org/en/20/):</b>
+- <b>log_queries:</b>False, meaning that SQL queries won't be logged. True, [echo flag](https://docs.sqlalchemy.org/en/20/core/engines.html#more-on-the-echo-flag) is set to true.
+- <b>create_db:</b> Create all tables stored in this project-defined entities metadata.[1](https://docs.sqlalchemy.org/en/20/core/metadata.html#sqlalchemy.schema.MetaData.create_all)
+- <b>session:</b> Configuration options related to SQLAlchemy sessions, which represent a "workspace" for interacting with the database.
+- <b>pool_recycle:</b> Specifies the number of seconds after which a connection will be recycled. In this case, connections will be recycled every 3600 seconds (1 hour).
+- <b>pool_pre_ping:</b> Control whether SQLAlchemy will "ping" the database before giving a connection from the pool to check if it's still valid. It's set to false, meaning that pre-pinging is disabled.
+- <b>url:</b> Define the database connection URL. It might be a single URL or a URL structure defined.
+```
+data:
+  sqlalchemy:
+    log_queries: false
+    url:
+      protocol: postgresql
+      username: ${oc.env:POSTGRES_USER}
+      password: ${oc.env:POSTGRES_PASSWORD}
+      host: ${oc.env:POSTGRES_HOST}
+      port: ${oc.env:POSTGRES_PORT}
+      file: ${oc.env:POSTGRES_DB}
+```
+  - <b>protocol:</b> Specifies the protocol used for the database connection. In this case, it's set to sqlite, indicating the usage of SQLite as the database backend.
```

Comment:
instead of specifying `protocol`. try to say in general this will build a url 
+ 
take short definition from sqlalchemy + link 
https://docs.sqlalchemy.org/en/20/core/engines.html